operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/EventSource.java

```diff
@@ -12,21 +12,15 @@
  */
 public interface EventSource extends LifecycleAware {
 
-  /**
-   * An optional name for your EventSource. This is only required if you need to register multiple
-   * EventSources for the same resource type (e.g. {@code Deployment}).
-   *
-   * @return the name associated with this EventSource
-   */
-  default String name() {
```

Comment:
Why do we have separate named event source now?

Reply:
Because names need to be configurable from outside the code. If it's part of the `EventSource` interface then the situation could happen that the annotation says one thing but the implementation specifies a different name, which would be error-prone.

Reply:
oki, makes sense!